server/src/services/etablissement.service.ts

```diff
@@ -172,6 +172,11 @@ export const getEtablissementFromGouv = async (siret: string): Promise<IAPIEtabl
 
     return data
   } catch (error) {
+
+    if(error.response.status == "404" || error.response.status == "422") {
```

Comment:
```suggestion
    if(["404", "422"].includes(error.response.status)) {
```

C'est pareil, mais ca évite la répétition de la condition.